config/index.ts

```diff
@@ -81,11 +81,11 @@ export function writeConfig(options: WriteConfigOptions = {}): void {
   }
 }
 
-export function getConfigPath(path?: string): string | null {
-  if (CLIConfiguration.isActive()) {
+export function getConfigPath(useNewConfig = false): string | null {
+  if (useNewConfig || CLIConfiguration.isActive()) {
     return getConfigFilePath();
   }
-  return config_DEPRECATED.getConfigPath(path);
+  return config_DEPRECATED.getConfigPath();
```

Comment:
You can probably update the deprecated config and remove this arg now that its not being used anywhere

Reply:
There is still one method in `config_DEPRECATED` that uses this function with the path argument: `loadConfigFromFile`. Going to keep as is for now but let me know if you have any ideas/thoughts.